itou/www/stats/views.py

```diff
@@ -99,11 +100,19 @@ def get_params_for_whole_country():
 
 
 def render_stats(request, context, params={}, template_name="stats/stats.html"):
+    view_name = get_view_name(request)
+    tally_form_id = settings.METABASE_DASHBOARDS[view_name].get("tally_form_id")
+    enable_tally_form = settings.ENABLE_TALLY_FORMS and tally_form_id is not None
```

Comment:
`is not None` est très explicite. J'aurais été plus fainéant :) (`not tally_form_id`)

Reply:
`not tally_form_id` est OK pour pylint alors que l'autre non il me semble bien.

Reply:
Si je résume, la liste des dashboards contient désormais : 
- un `dict` pour chaque entrée,
- contenant un ID, et un tally form ID

Reply:
Un classique : https://stackoverflow.com/questions/7816363/if-a-vs-if-a-is-not-none